packaging/datadog-agent/source/install_agent.sh

```diff
@@ -143,13 +149,19 @@ else
     $sudo_cmd sh -c "sed 's/api_key:.*/api_key: $apikey/' /etc/dd-agent/datadog.conf.example > /etc/dd-agent/datadog.conf"
 fi
 
-printf "\033[34m* Starting the Agent...\n\033[0m\n"
+restart_cmd="$sudo_cmd /etc/init.d/datadog-agent restart" 
 if command -v invoke-rc.d >/dev/null 2>&1; then
-    $sudo_cmd invoke-rc.d datadog-agent restart
-else
-    $sudo_cmd /etc/init.d/datadog-agent restart
+    restart_cmd="$sudo_cmd invoke-rc.d datadog-agent restart"
 fi
 
+if $no_start; then
+    printf "\033[34m* DD_NO_START environment variable set : the newly installed version of the agent will not start by itself. You will have to do it manually using the following command : \n\n $restart_cmd \n  \n\033[0m\n"
```

Comment:
Could you break it on multiple lines of max 80 chars ? (it's really better on terminal)
Could you also remove the space before the `:`: http://www.wikiwand.com/en/Colon_(punctuation)#/Spacing